### PR TITLE
Check if Mail is running will apple script

### PR DESCRIPTION
**Pull request by larkov**
Fixes issue #68

### CODE REVIEW

File: Packaging/preinstall [hunk 1]
```diff
@@ -4,7 +4,7 @@ temp=/private/tmp/MailTrackerBlocker-Installation-Temp
 running="$temp/running"
 mkdir -p "$temp"
 
-if [ `/usr/bin/pgrep Mail` ]; then
+if [ $(osascript -e $'if application "Mail" is running then\nreturn "True"\nelse\nreturn "False"\nend if') == "True" ]; then
```
Comment (apparition47) [inline Packaging/preinstall:7]: This should work just as well

```suggestion
if [ $(osascript -e 'return application "Mail" is running') == "true" ]; then
```

**Comment by apparition47** [comment]
Thanks for the PR! I was wondering why my Mail doesn't always restart reliably 🤔

**Comment by larkov** [comment]
Changed to check if the application id "com.apple.Mail" is running if it's called something else than "Mail" in some regions or if the user has renamed the app.

**Review by apparition47 — APPROVED** [review]
Thanks!

**Pull request merged**